llvm/test/tools/llc/llc_emit_spirv_val_error_as_warning.ll

```diff
@@ -0,0 +1,38 @@
+// REQUIRES: spirv-val,system-linux
```

Comment:
This test has dependency on spirv-val which is a SPIR_V validator tools from the Khronos SPIRV-Tools repo. Nt sure how to handle this dependency. Any comments will be helpful. Thanks